### ISSUE TITLE
Stale title after resume

### ISSUE THREAD

**Issue by jimmeta** [CyberShadow/term-title#2]
If you suspend emacs with ^Z, then the shell writes a new title, then you resume with `fg`, `term-title` won't know to replace that title.  This is a workaround:
```
(add-hook 'suspend-resume-hook
          (lambda ()
            (if (frame-live-p (selected-frame))
                (set-frame-parameter (selected-frame) 'term-title-last-title nil))))
```

**Comment by CyberShadow** [comment]
Makes sense to me. I don't use this functionality (I use dedicated terminals for Emacs so that I can use [term-keys](https://github.com/CyberShadow/term-keys) there), but feel free to put this in a pull request.
